src/main/scala/dpla/ingestion3/enrichments/EnrichmentDriver.scala

```diff
@@ -0,0 +1,60 @@
+package dpla.ingestion3.enrichments
+
+import dpla.ingestion3.model._
+
+
+class EnrichmentDriver {
+  val stringEnrich = new StringEnrichments()
+  val dateEnrich = new ParseDateEnrichment()
+
+  /**
+    * Applies a set of common enrichments that need to be run for all providers
+    *   * Spatial
+    *   * Language
+    *   * Type
+    *   * Date
+    *
+    * @param record The mapped record
+    * @return An enriched record
+    */
+  def enrich(record: DplaMapData): DplaMapData = {
```

Comment:
what @AudreyAltman  said... you should only have to specify the fields you're going to change. You might need to make a new DplaMapData, though, but that only has 3 fields.